src/test/java/us/kbase/test/auth2/lib/storage/mongo/MongoStorageTokensTest.java

```diff
@@ -56,11 +58,64 @@ TokenType.LOGIN, id, new UserName("bar"))
 					.withCustomContext("k1", "v1")
 					.withCustomContext("k2", "v2")
 					.build())
+			.withMfa(MfaStatus.NOT_USED)
 			.withTokenName(new TokenName("foo")).build();
 		final StoredToken st = storage.getToken(new IncomingToken("sometoken").getHashedToken());
 		assertThat("incorrect token", st, is(expected));
 	}
-	
+
+	@Test
```

Comment:
For consolidation purposes I'm repeating a comment from a previous review here - I think rather than making 2 new tests existing tests could be edited to use different mfa statuses